examples/00-fluent/exhaust_system_settings_api.py

```diff
@@ -535,13 +535,6 @@
 )
 volume_mesh_gen.Execute()
 
-###############################################################################
-# Check mesh
-# ~~~~~~~~~~
-# Check the mesh.
-
-meshing.tui.mesh.check_mesh()
```

Comment:
If we are removing meshing.tui calls as well, then we shoul provide an alternative for mesh check, etc. Perhaps a scheme call?

Reply:
@hpohekar  We have to retain TUI calls on the meshing side where there is no alternative. Scheme calls are an inferior alternative.

Reply:
Done. Thank you.